pkg/utils/utils.go

```diff
@@ -302,6 +304,30 @@ func getNetdevMTU(pciAddr string) int {
 	return mtu
 }
 
+func getNetDevMac(ifaceName string) string {
```

Comment:
Using the pci address is more reliable, as sometimes the interface is available.

Reply:
how can you get MAC address with pci address?

Reply:
You can also find the mac address from the sysfs. for example, /sys/bus/pci/devices/0000:01:00.0/net/eno1/address. However it would still require the interface name. So let keep it as it is.